### ISSUE TITLE
500 internal server error

### ISSUE THREAD

**Issue by specs32** [chubin/rate.sx#70]
`curl eur.rate.sx/xmr`

returns 

`<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request.  Either the server is overloaded or there is an error in the application.</p>
`

**Comment by chubin** [comment]
Please try again

**Comment by chubin** [comment]
Yes, it is because of `eur`. It does not work for the moment indeed. This need to be fixed.

**Comment by specs32** [comment]
Thanks for looking into this and Good luck !

**Comment by Hallavera** [comment]
Please look into this :(. We have been waiting 3 years to measure our europoorness in the right currency.
